package.json

```diff
@@ -9,12 +9,13 @@
     "url": "git@github.com:resin-io-modules/contrato.git"
   },
   "scripts": {
+    "buildtypes": "npx ts-node@^8.0.0 ./scripts/build-types.ts",
```

Comment:
Is `@^8.0.0` necessary?

Reply:
I was thinking to protect against breaking changes, but we aren't using any flags, so it should be fine to drop it